src/solvers/lowering/byte_operators.cpp

```diff
@@ -478,7 +517,33 @@ exprt lower_byte_extract(const byte_extract_exprt &src, const namespacet &ns)
       return simplify_expr(vector, ns);
     }
   }
-  else if(ns.follow(src.type()).id()==ID_struct)
+  else if(src.type().id() == ID_complex)
+  {
+    const complex_typet &complex_type = to_complex_type(src.type());
+    const typet &subtype = complex_type.subtype();
+
+    auto subtype_bits = pointer_offset_bits(subtype, ns);
+    if(subtype_bits.has_value())
+    {
+      byte_extract_exprt real(unpacked);
+      real.type() = subtype;
+
+      const plus_exprt new_offset(
```

Comment:
The coding standard says "Prefer brace style initialisation (i.e. type_name{arguments...}) over parentheses for constructor calls" :baby_chick:

Reply:
Done.